### DISCUSSION TITLE
order by (a / b)?

### DISCUSSION THREAD

**Discussion by izzysden** [prisma/prisma#18801] [Q&A] [answered]
I was trying to sort my table by a / b like a ratio or something, so I tried:
`SELECT * FROM user WHERE pulls > 0 ORDER BY tries / pulls`
and by this I could sort data by tries / pulls (tries per pull),

but in Prisma, how do I sort data by mathematical expression like `a / b`?

I tried:
`const data = await prisma.user.findMany({
        orderBy: {
          "tries / pulls": "asc",
        },
      });`
`const data = await prisma.user.findMany({
        orderBy: {
          temp: "tries / pulls",
        },
      });`
`const data = await prisma.user.findMany({
        orderBy: {
          tries: "/ pulls",
        },
      });`
but none of these worked.

**Comment by ludralph** [comment, marked as answer]
Hi @izzysden 👋 

We currently do not have support for sorting using a mathematical expression like a/b. You would need to use [raw queries](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#queryraw) to achieve this.
